tests/utils/helper.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import subprocess
+
+from ansible.parsing.dataloader import DataLoader
+from ansible.inventory.manager import InventoryManager
+
+
+def run_make_shell(args=None, tmpdir=None, **kwargs):
+    make_args = []
+    if tmpdir:
+        make_args.append('SHELL_ENV={}'.format(tmpdir))
+    if args:
+        command = args
+    else:
+        command = 'true'
+    make_args.append('C="{}"'.format(command))
+    full_command = 'make shell {args}'.format(args=' '.join(make_args))
+    logging.warning("Running: {}".format(full_command))
+    make_process = subprocess.Popen(full_command, shell=True, **kwargs)
+    make_process.wait()
+    return make_process
+
+
+def run_ansible_playbook(playbook, inventory=None, env=None, tmpdir=None):
+    env_vars = dict(os.environ)
+    env_vars.setdefault('ANSIBLE_FORCE_COLOR', "true")
```

Comment:
I'd rather set this in the environment (e.g. Eve's config) rather than here and force things.

Reply:
But ansible detect that output is not a tty and disable color. That's mainly for human

Reply:
Problem would be when running in CI environments which don't support colors in TTY, and would result in completely messed up job outputs. Looking at you, Jenkins!

Reply:
But jenkins supports color (with a plugin, if I remember correctly), buildbot has also support. And there still the option to export the variable ANSIBLE_FORCE_COLOR=false to disable color. For sure accurate detection would be a plus, but in the meantime, it's more convenient ?